apps/Gruntfile.js

```diff
@@ -880,8 +883,46 @@ describe('entry tests', () => {
         new StatsWriterPlugin({
           fields: ['assetsByChunkName', 'assets']
         }),
+        new CopyPlugin([
+          {
+            from: 'build/locales',
+            to: minify
+              ? '[path]/[name]wp[contenthash].[ext]'
+              : '[path]/[name].[ext]',
+            toType: 'template'
+          },
+          // Libraries in this directory are assumed to have .js and .min.js
+          // copies of each source file. In development mode, copy only foo.js.
+          // In production mode, copy only foo.min.js and rename it to foo.js.
+          // This allows the manifest to contain a single mapping from foo.js
+          // to a target file with the correct contents given the mode.
+          //
+          // Ideally, the target file would have the .min.js suffix in
+          // production mode. This could be accomplished by nesting these files
+          // within a minifiable-lib directory in the output package so that the
+          // manifest plugin could do special processing on these files.
```

Comment:
Sorry this comment wasn't very clear. The proposed future change would be to change the name of the file in the output bundle so that the names of minified files have .min in them, not to include more copies of the file.

In production mode today we have something like this in the manifest:
```
  "js/droplet/droplet-full.js": "/assets/js/droplet/droplet-fullwp0573b060df80c10bff8ba38a881f31a0.js"
```
if we took these extra steps, we could have something with `.min` in the path value like this:
```
  "js/droplet/droplet-full.js": "/assets/js/droplet/droplet-full.minwp0573b060df80c10bff8ba38a881f31a0.js"
```
This is slightly better because you can see that you're getting a minified resource when you look at what your browser is downloading, whereas with the currently proposed approach a developer may say "why am I getting non-minified droplet?" and spend some time investigating before discovering these comments.